react/features/base/ui/components/web/ContextMenu.tsx

```diff
@@ -119,6 +119,8 @@ interface IProps {
      * Tab index for the menu.
      */
     tabIndex?: number;
+
+    useEntityWidth?: boolean;
```

Comment:
Pl document this. Also, wouldn't something like `containerWidth` be more descriptive?